api/v1/ytsaurus_types.go

```diff
@@ -452,6 +452,8 @@ type UISpec struct {
 	Theme       string  `json:"theme,omitempty"`
 	Description *string `json:"description,omitempty"`
 	Group       *string `json:"group,omitempty"`
+	// When this is set to false, UI will use backend for downloading instead of proxy.
```

Comment:
As I mentioned here https://github.com/ytsaurus/yt-k8s-operator/pull/257#discussion_r1604919789 this comment should help users to choose the right value. Since it is a bool pointer we have three possible values here: nil, true, and false.
You have only documented only the `false` option, so when users update their operator they may have reasonable question: currently my spec doesn't have such field, so what happens when I update the operator with the same spec. It is not obvious to me what "direct" means here, so I suppose it may not be obvious for users (suppose if false means backend then proxy is the direct one). Usually `nil` and `false` values mean the same, so as a user, I would suggest that if I don't set the value I get UI with changed behaviour. But at the same time for some reason you documenting the only `false` option, so maybe (and I think that is the case) the default is `true`.

If my guess is right and this (poorly named in UI) option is true for default — there is more reason to document it clearly. So if true is really a default for UI config let us say something like:
```
When DirectDownload is true or omitted, UI uses the proxies for downloading, which is the default behaviour.
If set to false, UI uses its backend for downloading.
```

It all would be less confusing if settings was named `BackendDownload`, but since it is in UI public configuration let's stick with `DirectDownload` and not add entropy.